packages/fxa-auth-server/lib/routes/oauth/token.js

```diff
@@ -387,6 +392,31 @@ module.exports = ({ log, oauthDB, db, mailer, devices }) => {
           mode: 'optional',
           strategy: 'sessionToken',
         },
+        description: '🔒🔓 sessionToken',
+        notes: [
+          '<p>🔒🔓 Optionally HAWK-authenticated with session token</p><p>Grant new OAuth tokens for use by a connected client, using one of the following grant types:</p>' +
+            '<ul><li><code>grant_type=authorization_code</code>: A single-use code obtained via OAuth redirect flow.</li>' +
+            '<li><code>grant_type=refresh_token</code>: A refresh token issued by a previous call to this endpoint.</li>' +
+            '<li><code>grant_type=fxa-credentials</code>: Directly grant tokens using an FxA sessionToken.</li></ul>' +
+            '<p>This is the "token endpoint" as defined in RFC6749, and behaves like the (oauth-server /token endpoint)[../fxa-oauth-server/docs/api.md#post-v1token] except that the <code>fxa-credentials</code> grant can be authenticated directly with a sessionToken rather than with a BrowserID assertion.</p>',
```

Comment:
Same here, the brackets and parens are backwards.
```suggestion
            '<p>This is the "token endpoint" as defined in RFC6749, and behaves like the [oauth-server /token endpoint](../fxa-oauth-server/docs/api.md#post-v1token) except that the <code>fxa-credentials</code> grant can be authenticated directly with a sessionToken rather than with a BrowserID assertion.</p>',
```